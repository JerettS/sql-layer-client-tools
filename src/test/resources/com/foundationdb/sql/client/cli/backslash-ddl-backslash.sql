\lt test.foo
CREATE TABLE foo(id INT);
\lt test.foo
