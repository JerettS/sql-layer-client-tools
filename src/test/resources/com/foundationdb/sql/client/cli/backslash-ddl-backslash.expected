Schema | Name 
--------+------
(0 rows)

ROWS: 0

 Schema | Name 
--------+------
 test   | foo  
(1 row)
